lib/dependencies.sh

```diff
@@ -31,14 +31,16 @@ run_if_present() {
 
 yarn_node_modules() {
   local build_dir=${1:-}
+  $NODE_VERBOSE && local opt_verbose='--verbose' || local opt_verbose=''
```

Comment:
I think `NODE_VERBOSE` is too generic. Can we switch it to something more explicit? 

Suggestion: `HEROKU_BUILD_VERBOSE`
We could then standardize on that across buildpacks, add a stdlib call to log verbose things.